Library/Homebrew/formula_installer.rb

```diff
@@ -85,7 +85,7 @@ def pour_bottle?(install_bottle_options = { warn: false })
     return false if @pour_failed
 
     bottle = formula.bottle
-    return false unless bottle
+    return false unless bottle || formula.local_bottle_path
```

Comment:
`return true  if formula.local_bottle_path` is on line 94 already so it feels weird to duplicate it here. Can this logic be moved around instead so it's only checked once?

Reply:
I've removed `return true if formula.local_bottle_path`. Pouring a local bottle should still check that that bottle is compatible. `--force-bottle` may be used to pour a local bottle that is incompatible, as it is for remote bottles.

Reply:
Ok, cool. Can you make this an `if` instead; `unless` with multiple conditions are hard to parse. After that should be 👍 to 🚢

Reply:
Done. Thanks for the feedback, Mike.